pkg/cloudprovider/controller/nodecontroller_test.go

```diff
@@ -721,8 +823,8 @@ func TestSyncNodeStatusTransitionTime(t *testing.T) {
 				Status: probe.Success,
 				Err:    nil,
 			},
-			expectedRequestCount:         2, // List+Update
-			expectedTransitionTimeChange: false,
+			expectedRequestCount:   2, // List+Update
+			expectedTransitionTime: util.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 		},
 		{
 			// Existing node is healthy, current probe is unhealthy.
```

Comment:
Can you add a comment why you get api.ConditionFull instead of api.ConditionUnknown, even though the current probe is unhealthy? I assume it is because the timeout has not passed yet.